ipv8/messaging/interfaces/endpoint.py

```diff
@@ -129,6 +129,7 @@ def __init__(self, endpoint, main_thread=True):
         self.endpoint = endpoint
 
         self._netifaces_failed = netifaces is None
+        self._local_interfaces = []
```

Comment:
This was already included in #999. Generally: please make sure to only make one change per PR.

Reply:
Oh whoops, this one slipped through I guess 🤔